sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/ParallelTransferOptions.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.common;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.storage.common.implementation.StorageImplUtils;
+
+/**
+ * This class contains configuration used to parallelize data transfer operations. Note that not all values are used
+ * by every method which accepts this type. Please refer to the javadoc on specific methods for these cases.
+ */
+@Fluent
+public final class ParallelTransferOptions {
+
+    private final Integer blockSize;
+    private final Integer numBuffers;
+    private final ProgressReceiver progressReceiver;
+    private final Integer maxSingleUploadSize;
+
+    /**
+     * Creates a new {@link ParallelTransferOptions} with default parameters applied.
+     *
+     * @param blockSize The block size.
+     * For upload, The block size is the size of each block that will be staged. This value also determines the number
```

Comment:
Is this also the value we use for chunks in downloadToFile? Perhaps we should update these docs

Reply:
I added a sentence for downloadToFile